x-pack/plugins/security_solution/server/lib/siem_migrations/rules/api/create.ts

```diff
@@ -47,7 +47,7 @@ export const registerSiemRuleMigrationsCreateRoute = (
               migration_id: migrationId,
               original_rule: originalRule,
             }));
-
+            await ruleMigrationsClient.data.integrations.create();
```

Comment:
For some reason, this is still creating duplicate documents, I would have liked this to run once, or if its run multiple times, at least override the existing documents, but I might be missing a parameter in the bulk create function..

Reply:
If you want to "upsert" those documents, you need to call use the `update` operation with `doc_as_upsert: true`, check out how we do it for resources:

https://github.com/elastic/kibana/blob/b24fdf5d3f6b7454a4edcedb8141b82f571e1d74/x-pack/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_resources_client.ts#L28-L35

Reply:
That is weird, as `op_type` `index` should `create or update if exists`, but I am more than happy to change to `update`. I assume then it will create if the ID does not exist?

Reply:
exactly